### PR TITLE
Get appointment request

### LINKED ISSUE #39: GET request for appointments by mentor id
#### Description
Create a GET request that returns all of the requested appointments for a specific mentor given the mentor id. 
Dependent on PR #31 
#### Files
`backend/api/views/appointment.py`
(Create if file does not exist yet)

#### Tasks
- [ ] Determine request name, params if necessary, etc.
- [ ] Write the request!
- [ ] Test it on Postman (check if there's anything in the db first with MongoDB Compass for expected results, and make sure it's not breaking the backend!)
- [ ] Create a PR with ^ as screenshots and a description of what you've completed!

### PR DESCRIPTION
**Pull request by nroy20**
## Description :star2:

Created a GET request that returns all requested appointments for the specified mentor given their id.
Fixes #39 
 
## TODOs :star:
- [x] Tests
- [x] Documentation
- [x] Move to appointments file in view (dependent on #46)
- [ ] Reviews

### CODE REVIEW

File: backend/api/views/main.py [hunk 1]
```diff
@@ -35,6 +35,18 @@ def get_mentor(mentor_id):
     return create_response(data={"mentor": mentor})
 
 
+# GET request for appointments by mentor id
+@main.route("/requests/<string:mentor_id>", methods=["GET"])
```
Comment (chyku) [inline backend/api/views/main.py:39]: wanted to note that we want this in a separate file for requests (another file in views that someone else is adding)

File: backend/api/views/appointment.py [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 appointment = Blueprint("appointment", __name__)
 
+# GET request for appointments by mentor id
+@appointment.route("/appointment/<string:mentor_id>", methods=["GET"])
+def get_requests(mentor_id):
```
Comment (chyku) [inline backend/api/views/appointment.py:11]: can this be clearer that it gets appointments by mentor ID in the endpoint URL & request function name (as opposed to appointment request ID or something)

File: backend/api/views/main.py [hunk 3]
```diff
@@ -35,6 +43,20 @@ def get_mentor(mentor_id):
...
+    try:
+        request = MentorProfile.objects.get(id=appointment_id)
+        request.delete()
+    except:
+        msg = "The request you attempted to delete was not found"
+        logger.info(msg)
+        return create_response(status=422, message=msg)
+    return create_response(data={"deleted id": appointment_id})
+"""
+
```
Comment (chyku) [inline backend/api/views/main.py:59]: lol why is this in this PR

**Review by chyku — CHANGES_REQUESTED** [review]
idk if this is ready bc the pr desc still has in dev but i have comments LOL

**Review by chyku — APPROVED** [review]
:heart_decoration:

File: backend/api/views/appointment.py [hunk 4]
```diff
@@ -5,6 +5,16 @@
...
+# GET request for appointments by mentor id
+@appointment.route("/appointment/mentor/<string:mentor_id>", methods=["GET"])
+def get_requests_by_mentor(mentor_id):
+    requests = AppointmentRequest.objects(mentor_id=mentor_id)
+    return create_response(data={"requests": requests})
+
+
+@appointment.route("/appointment/accept/<id>", methods=["PUT"])
+def put_appointment(id):
+=======
```
Comment (chyku) [inline backend/api/views/appointment.py:17]: lol i think you merged this wrong

Comment (chyku) [reply backend/api/views/appointment.py:17]: if u delete this it's an insta approve

**Review by chyku — DISMISSED** [review]

**Review by Leundai — APPROVED** [review]

**Review by AngelaLuo49021 — APPROVED** [review]
Look good!

**Pull request merged**